### PR TITLE
feat: details page for disk images

### LINKED ISSUE #856: Add a Disk Image details page
### Is your enhancement related to a problem? Please describe

Bootable containers don't have a details page.

### Describe the solution you'd like

We should add a details page for each bootable container:
- More consistent with Podman Desktop's patterns
- A place to put additional information (don't need to add columns to the table for everything)
- Future place to add terminal window in support of features like #752.
- We're already doing it for the build logs; will fit in better.

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by deboer-tim**
### What does this PR do?

Modifies the build logs to have a proper details page for disk images, similar to how Podman Desktop handles Images or Containers.

The two tabs are Summary and Build Logs. The summary page is simple for now but could be expanded; the build logs doesn't really match what we typically have in Podman Desktop, but it fits in better here than a standalone page.

### Screenshot / video of UI

Before:

https://github.com/user-attachments/assets/1b6f3788-b3b2-49e3-8798-94bf19da9d19

After:

https://github.com/user-attachments/assets/8069ea15-bb7a-42b8-a86a-1fcd6655e576

### What issues does this PR fix or reference?

Fixes #856.

### How to test this PR?

Build some disk images, switch between homepage and disk image, use homepage action to go straight to a build.

### CODE REVIEW

File: packages/frontend/src/lib/disk-image/DiskImageDetails.svelte
Comment (axel7083) [inline packages/frontend/src/lib/disk-image/DiskImageDetails.svelte:22]:
```suggestion
```

File: packages/frontend/src/lib/disk-image/DiskImageDetails.svelte
Comment (axel7083) [inline packages/frontend/src/lib/disk-image/DiskImageDetails.svelte:25]:
```suggestion
```

File: packages/frontend/src/lib/upstream/Util.ts
Comment (axel7083) [inline packages/frontend/src/lib/upstream/Util.ts:2]:
```suggestion
 * Copyright (C) 2024 Red Hat, Inc.
```

File: packages/frontend/src/lib/upstream/Util.ts
Comment (axel7083) [inline packages/frontend/src/lib/upstream/Util.ts:30]: Some unit tests maybe ?

**Review by axel7083 — APPROVED** [review]
Tested locally on Windows 11 and works like a charm 🚀 LGTM (some comments added)

File: packages/frontend/src/lib/disk-image/DiskImageDetailsSummary.svelte
Comment (axel7083) [inline packages/frontend/src/lib/disk-image/DiskImageDetailsSummary.svelte:18]: Something nice (follow up PR) could be done here, is to have the image as a link and [redirect to the image](https://github.com/containers/podman-desktop/blob/208b2a07fc172213fc5b3ee143e463a24fbfe597/packages/extension-api/src/extension-api.d.ts#L4684-L4685) details page.

File: packages/frontend/src/lib/disk-image/DiskImageDetails.svelte
Comment (deboer-tim) [reply packages/frontend/src/lib/disk-image/DiskImageDetails.svelte]: Removed.

File: packages/frontend/src/lib/disk-image/DiskImageDetails.svelte
Comment (deboer-tim) [reply packages/frontend/src/lib/disk-image/DiskImageDetails.svelte]: Removed.

File: packages/frontend/src/lib/upstream/Util.ts
Comment (deboer-tim) [reply packages/frontend/src/lib/upstream/Util.ts:30]: I copied this from PD, where there were no tests 😬. I've added some, and will PR to PD as well.

File: packages/frontend/src/lib/disk-image/DiskImageDetailsSummary.svelte
Comment (deboer-tim) [reply packages/frontend/src/lib/disk-image/DiskImageDetailsSummary.svelte:18]: Good idea! Yes, I'll do this via another PR.

**Review by cdrage — APPROVED** [review]
Looks absolutely great and thanks for this change! 

After testing, the implementation looks great and I have no issues with the code / code LGTM.

I'm going to rebase #865 so that it's in a tab similar to this implementation after this is merged in.

**Pull request merged**
